### PR TITLE
Refine assertion text

### LINKED ISSUE #161: Refine `AssertionError` message for `NRowsMinGain` and `NRowsMaxGain` constraints
When the constraint `NRowsMinGain` is not met, the following exemplary `AssertionError` is thrown:

> AssertionError: TAB_NEW has 0.30231844350740233 of #rows of TAB_OLD. It was supposed to increase at least 0.5.

I find this message misleading as TAB_NEW has actually 130 % of #rows of TAB_OLD, or 30 % increase in #rows. 

As a minimal refinement I would suggest to replace the string "of #rows of" with "relative gain in #rows compared to" in the `assertion_text ` of `NRowsMinGain`.

This would change the message to
> AssertionError: TAB_NEW has 0.30231844350740233 relative gain in #rows compared to TAB_OLD. It was supposed to increase at least 0.5.

Additionally, one could express the numbers as percentage by multiplying `relative_gain` as well as `self.min_relative_gain` with 100, adding "%" to the string and omitting the "relative". That would give us:
> AssertionError: TAB_NEW has 30.231844350740233 % gain in #rows compared to TAB_OLD. It was supposed to increase at least 50 %.

Similarly, it would probably make sense to change the error message in `NRowsMaxGain`.

### PR DESCRIPTION
**Pull request by jak-ket**
Fix #161

### CODE REVIEW

**Comment by kklein** [comment]
@ivergara Do you have a clue why this unit test is failing?

**Comment by ivergara** [comment]
@kklein don't understand why the setup is failing to find `git`. I thought it might have been a fluke so I (in principle) retriggered the job but it seems to have failed too.

Will take a deeper look at some point.

File: src/datajudge/constraints/nrows.py [hunk 1]
```diff
@@ -115,9 +115,9 @@ def compare(self, n_rows_factual: int, n_rows_target: int) -> Tuple[bool, str]:
             return True, "Row loss."
         relative_gain = (n_rows_factual - n_rows_target) / n_rows_target
         assertion_text = (
-            f"{self.ref.get_string()} has {relative_gain} of #rows of "
+            f"{self.ref.get_string()} has {relative_gain:%} gain in #rows compared to "
             f"{self.ref2.get_string()}. It was only allowed "
-            f"to increase {self.max_relative_gain}. "
+            f"to increase {self.max_relative_gain:%}. "
```
Comment (kklein) [inline src/datajudge/constraints/nrows.py:120]:
```suggestion
            f"to increase by {self.max_relative_gain:%}. "
```

File: src/datajudge/constraints/nrows.py [hunk 2]
```diff
@@ -146,9 +146,9 @@ def compare(self, n_rows_factual: int, n_rows_target: int) -> Tuple[bool, str]:
             return False, "Row loss."
         relative_gain = (n_rows_factual - n_rows_target) / n_rows_target
         assertion_text = (
-            f"{self.ref.get_string()} has {relative_gain} of #rows of "
+            f"{self.ref.get_string()} has {relative_gain:%} gain in #rows compared to "
             f"{self.ref2.get_string()}. It was supposed "
-            f"to increase at least {self.min_relative_gain}. "
+            f"to increase at least {self.min_relative_gain:%}. "
```
Comment (kklein) [inline src/datajudge/constraints/nrows.py:151]:
```suggestion
            f"to increase at least by {self.min_relative_gain:%}. "
```

**Review by kklein — APPROVED** [review]
Looks way better than before!

**Comment by kklein** [comment]
Thank you @jak-ket !

**Pull request merged**